multiqc/templates/default/assets/js/multiqc_toolbox.js

```diff
@@ -4,6 +4,7 @@
 
 var mqc_colours_idx = 0;
 var mqc_colours = ["#e41a1c", "#377eb8", "#4daf4a", "#984ea3", "#ff7f00", "#ffff33", "#a65628", "#f781bf", "#999999"];
+var zip_threshold = 1;
```

Comment:
Let's only use zip files when we really have to..

```suggestion
var zip_threshold = 8;
```